### PR TITLE
connectors/package-lock.json  was not properly updated when updating types/package.json

### PR DESCRIPTION
**Pull request by lasryaric**
## New description

The main goal of this PR is to fix `connectors/` in the CI to be able to land [another PR](https://github.com/dust-tt/dust/pull/5434) that is going to fix a Datadog monitor.

Some changes in types/ were not properly CI checked and broke connectors.
Fixing connectors actually involves reverting some changes in types/, and not fixing connectors.

## Initial description

Updating `connectors/package.json`.
When updating types/package-lock.json, we need to run `npm install` in `connectors/`, otherwise this leads to inconsistency. 
This PR fixes an inconsistency added by [this](https://github.com/dust-tt/dust/pull/5406) PR, which added `moment-timezone` to types/

### CODE REVIEW

**Review by flvndvd — APPROVED** [review]

**Comment by lasryaric** [comment]
@flvndvd you are not required to review anymore, but I can't review you since you already approved.

File: types/src/front/membership_invitation.ts [hunk 1]
```diff
@@ -16,7 +15,7 @@ export type MembershipInvitationType = {
 // Types for the invite form in Poke.
 
 export const InviteMemberFormSchema = t.type({
-  email: NonEmptyString,
+  email: t.string,
   role: ActiveRoleSchema,
```
Comment (JulesBelveze) [inline types/src/front/membership_invitation.ts:19]: Heads up: this wasn't introduced by #5406 dunno if you wanna change it to `t.string`

**Review by JulesBelveze — COMMENTED** [review]
Thx 🙏🏼

**Review by JulesBelveze — APPROVED** [review]
LGTM

**Pull request merged**